spec/rails_helper.rb

```diff
@@ -75,3 +75,19 @@
 def open_structify(data)
   JSON.parse(data.to_json, object_class: DatedStruct)
 end
+
+def stub_call_to_get_json_schema
+  stub_request(:get, 'http://localhost:3000/schemas/assessment_request.json')
+    .with(
+      headers: {
+        'Accept' => '*/*',
+        'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+        'User-Agent' => 'Ruby'
+      }
+    )
+    .to_return(status: 200, body: full_schema, headers: {})
```

Comment:
I don't think the headers need to be defined for this stub to work, and their size implies they are important when they are not.
```ruby
stub_request(:get, 'http://localhost:3000/schemas/assessment_request.json')
  .to_return(status: 200, body: full_schema)
```